lib/Support/ErrorHandling.cpp

```diff
@@ -38,6 +38,11 @@ cl::opt<bool> WarningsOnlyToFile(
     "warnings-only-to-file", cl::init(false),
     cl::desc("All warnings will be written to warnings.txt only.  If disabled, "
              "they are also written on screen."));
+
+cl::opt<bool> LogCpuTime(
```

Comment:
I also thought about making this an option, but dismissed that, as there already that many.